db/data_migrate/20190212163200_remove_submissions_with_no_task.rb

```diff
@@ -0,0 +1,20 @@
+# - Remove submission `c06c10ed-ae0c-40ee-b8ee-e8287e4882ab` (supplier was the last person
+#   submitting on CM/OSG/05/3565, which has been retired).
+# - Remove submission `628ac96c-2c8d-4beb-b4ff-b3fc185b739e` (supplier has been removed from
+#   DOS3, and therefore the associated task was deleted).
+#
+# Execute with:
+#
+#   rails runner db/data_migrate/20190212163200_remove_submissions_with_no_task.rb
+#
+
+RETIRED_FRAMEWORK_SUBMISSION = 'c06c10ed-ae0c-40ee-b8ee-e8287e4882ab'.freeze
+ERRONEOUS_DOS3_SUBMISSION    = '628ac96c-2c8d-4beb-b4ff-b3fc185b739e'.freeze
+
+Submission.delete(RETIRED_FRAMEWORK_SUBMISSION)
```

Comment:
I think I'd prefer `destroy` instead of `delete`, so we can be certain that any callbacks were executed, but I suppose in this specific case it doesn't matter much because we "know" there aren't. (And the thing preventing us from destroying a submission with associated files and/or entries is at db level, not Rails level.)